packages/app/src/shared/components/token-super-field/token-super-field.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from 'react'
+import { SuperField } from '@marginly/ui/components/input/super-field'
+import { useFocus } from '@marginly/ui/components/input/use-focus'
+
+const NUMBER_INPUT_REGEX = /^[0-9]+(?:\.[0-9]*)?$/
```

Comment:
As a requirement, we also should support values like `.2`

Reply:
Will be fixed in the next version

Reply:
Ok